### PR TITLE
Jomarton/pagination

### PR DESCRIPTION
**Pull request by JoMarton**
updates to the pagination component:
- changing DOM structure
- accessibility

### CODE REVIEW

File: packages/wix-ui-core/package.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
     "react-test-renderer": "^15.5.4",
     "ts-jest": "^21.2.4",
     "typescript": "^2.6.2",
-    "wix-ui-test-utils": "^1.0.0"
+    "wix-ui-test-utils": "^1.0.16"
```
Comment (amiryonatan) [inline packages/wix-ui-core/package.json]: Leave it with 1.0.0 because otherwise it screws with lerna

**Comment by lbelinsk** [comment]
@JoMarton since @amiryonatan approved, you can merge it, but please read my comment about rtl. I think we should go with this approach, at least at the moment.

**Comment by JoMarton** [comment]
@lbelinsk my apologies, I wanted to merge and forgot to pull the trigger :). I don't see a comment from about rtl though... am I missing something?

**Comment by lbelinsk** [comment]
lol, I also forgot to click the submit review button :)
now you should see my comment

File: packages/wix-ui-core/src/components/Pagination/Pagination.tsx [hunk 2]
```diff
@@ -33,28 +37,43 @@ class Pagination extends React.Component<PaginationProps, PaginationState> {
...
+    /** Text to appear for the 'first' navigation button when prop 'replaceArrowsWithText' is true */
+    firstText: PropTypes.string,
+    /** Text to appear for the 'previous' navigation button when prop 'replaceArrowsWithText' is true */
+    previousText: PropTypes.string,
+    /** Text to appear for the 'next' navigation button when prop 'replaceArrowsWithText' is true */
+    nextText: PropTypes.string,
+    /** Text to appear for the 'last' navigation button when prop 'replaceArrowsWithText' is true */
+    lastText: PropTypes.string,
+    /** The pagination direction. acceots 'ltr' (left to right) or 'rtl' (right to left). defaults to 'ltr' */
+    direction: PropTypes.oneOf(['ltr' , 'rtl']),
```
[the remark this answers is not in the document]
Comment (JoMarton) [reply packages/wix-ui-core/src/components/Pagination/Pagination.tsx]: ok will do

**Pull request merged**